doc/source/man.rst

```diff
@@ -32,206 +32,217 @@ rho performs four major tasks:
 
 * Creating authentication profiles:
 
-``rho auth add ...``
+  ``rho auth add ...``
 
 * Creating network profiles:
 
-``rho profile add --name=X --hosts X Y Z --auth A B``
+  ``rho profile add --name=X --hosts X Y Z --auth A B``
 
 * Running a scan:
 
-``rho scan --profile=X --reportfile=Y``
+  ``rho scan --profile=X --reportfile=Y``
 
 * Working with facts that are gathered in a scan:
 
-``rho fact ...``
+  ``rho fact ...``
 
 The following sections describe these commands, their subcommands, and their options in more detail.
 
-Authentication
---------------
+Authentication Profiles
+-----------------------
 
-The first part to configuring rho is setting up authentication credentials. rho uses SSH to connect to the servers on the network, and the username and password or ssh key are used for authentication credentials. An authentication credential is passed when the scan is run by reference in a profile.
+Use the ``rho auth`` command to create and manage authentication profiles.
 
-There can be multiple auth credentials contained in a single profile.
+An authentication profile defines a set of user credentials to be used during a scan. These user credentials include a username and a password or SSH key. Rho uses SSH to connect to servers on the network and uses authentication profiles to obtain the user credentials that are required access those servers.
```

Comment:
Does `that are required to access those servers` sounds better?

Reply:
Yes, thanks. Missed that missing "to" in this morning's re-read of the info.

Reply:
Change made.